share/scripts/inverse/functions_common.sh

```diff
@@ -642,22 +642,22 @@ add_to_csgshare() { #added an directory to the csg internal search directories
   for dirlist in "$@"; do
     old_IFS="$IFS"
     IFS=":"
```

Comment:
Well, I was surprised myself, but without these changes (${var} vs £var) I was getting errors and could not progress with my testing. Sorry I included them now, in this commit, but otherwise it did not work for me.